loopy/transform/iname.py

```diff
@@ -752,26 +776,36 @@ def untag_inames(
 
 # {{{ tag inames
 
-_Tags_ish: TypeAlias = Tag | Collection[Tag] | str | Collection[str]
+_Tags_ish: TypeAlias = Tag | Collection[Tag] | str | Collection[str] | None
 
 
 @for_each_kernel
 def tag_inames(
             kernel: LoopKernel,
-            iname_to_tag: (Mapping[str, _Tags_ish]
+            iname_to_tag: (
+                Mapping[str, _Tags_ish]
                 | Sequence[tuple[str, _Tags_ish]]
                 | str),
+            *,
+            # FIXME: 'force' is unused?
```

Comment:
Kill it and see what breaks? Or at least deprecate?

Reply:
Added a deprecation warning. Seems to have been unused for a long time now: https://github.com/inducer/loopy/commit/8734898181feb996ca635a16726f08467535f923#diff-a91a39471a27d37ceb2117f72be493a2ab46af1987edc52ff7a432e2c296f2cc.